ads/google/a4a/line-delimited-response-handler.js

```diff
@@ -22,7 +22,7 @@ import {tryParseJson} from '../../../src/json';
  * @param {!Window} win
  * @param {!Response} response
  * @param {function(string, boolean)} lineCallback
- * @private
+ * @protected
```

Comment:
These should be `@package`, no? Unless they're `@public`.

Reply:
Making this `@package` results in this error:

```js
extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js:1511: ERROR - [JSC_BAD_PACKAGE_PROPERTY_ACCESS] Access to package-private property lineDelimitedStreamer$$module$ads$google$a4a$line_delimited_response_handler of ads/google/a4a/line-delimited-response-handler.js not allowed here.
            lineDelimitedStreamer(this.win, response, slotCallback);
            ^^^^^^^^^^^^^^^^^^^^^
```